views/diner.pug

```diff
@@ -5,42 +5,97 @@ block style
         nameplate {
            float: left;
            width: 35%;
-           height: 500px;
            text-align: center;
            border-style: solid;
            border-width: 0.05px;
-           border-color: #EC3017;
+           border-color: #FECA1D;
            padding: 20px;
         }
         information {
            float: right;
            padding: 20px;
            width: 60%;
-           height: 500px;
+           height: 1000px;
+        }
+        divider {
+           float: left;
+           width: 33.33%;
+        }
+        clearfix:after {
+          content: "";
+          clear: both;
+          display: table;
         }
 
 block content
   <section>
         <nameplate>
-            h4 Profile
+            img(src='../images/logo.png' style='width:50px; height:50px;')  
+            br
+            br
+            h4 My Profile
+            br
+            p #{diner.name}
+            br
+            if user.uname === diner.uname
+                a.btn.btn-primary(href='/diners/logout') Log Out
+                br
+                br
+                a.btn.btn-danger.delete-diner(href="#" usr=diner.uname) Delete Account
+            br
         </nameplate>
+        <clearfix>
         <information>
             h2 Hello, #{diner.uname}!
             br
             hr
-            img(src='../images/points_icon.png' style='width:25px; height:25px; float:left;')
+            img(src='../images/points_icon.png' style='width:25px; height:25px; float:left; margin-right: 10px;')
             p <b>Total Points </b> #{points.count}
+            img(src='../images/restaurant_icon.png' style='width:25px; height:25px; float:left; margin-right: 10px;')
             p <b>Most Visited Restaurants</b>
+            if visited.length == 0
```

Comment:
use `===` instead of `==`

Reply:
for all the `==` should use `===` instead haha

https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a